ui/src/app/app-routing.module.ts

```diff
@@ -131,15 +135,16 @@ const routes: Routes = [
                       // use the (urldecoded) captured value nbName
                       pathElementForTitle: 'nbName',
                       breadcrumb: BreadcrumbType.Notebook,
-                      minimizeChrome: true
+                      minimizeChrome: true,
+                      helpContent: 'data'
                     }
                   }, {
                     path: 'preview/:nbName',
                     component: InteractiveNotebookComponent,
                     data: {
                       pathElementForTitle: 'nbName',
                       breadcrumb: BreadcrumbType.Notebook,
-                      minimizeChrome: true
+                      minimizeChrome: true,
```

Comment:
I think this is an unrelated change? Should revert unless we're planning to have the helpContent here? (Which may make sense. I don't know what the mocks look like.)